docs/proposals/Avatar/README.md

```diff
@@ -0,0 +1,78 @@
+# Avatar
+
+An `<Avatar />` component will be used to display an identifier (image or
+initials) for a user.
+
+Rebecca Li has done some
+[great research](https://jobber.atlassian.net/browse/JOB-19028) on the usage of
+avatars in Jobber, as well as the potential usage of colors within avatars.
+
+## Interface
+
+```
+<Avatar
+    image="ImageUrl"
+    name="Eddy Sims"
+    initials="ES"
+    color="#8f2d56" | color="var(--color-red)"
+    size="large" | size={32}
+/>
+```
+
+## Props Table
+
+| name     | type   | default  | description                               |
+| -------- | ------ | -------- | ----------------------------------------- |
+| image    | String | -        | The avatar image to be use                |
+| name     | String | -        | A users name to be used for accessibility |
+| initials | String | -        | Initials to show if the image is null     |
+| color    | String | greyBlue | The border color that represents the user |
+| size     | String | small    | Size of the avatar - 'small'              | 'large' |
+
+## Display
+
+An `<Avatar />` component would have a heiarchy of display options
+
+1. Display the image used in the `image` prop.
+2. If `image` is null, display the `initials` prop.
+   - Initials should fit within the avatar and scale with the size.
+   - If initials contain 3 letters, the font size should scale to proplerly fit
+     this
+3. If `image` and `initials` are null. Display a fallback icon
+   - We could use `person` from https://atlantis.getjobber.com/components/icon
+
+## Color
+
+Can be used to represent the border color of the avatar if an image is set, or
+the background color of the avatar if the image is not set.
+
+One thing to note is that for accessibility purposes, color selection could
+cause issues with color contrast.
+
+## Size
+
+There are 2 options that I can think of for showing size, numeric and string.
+
+- Numeric: `size` prop takes a number and that is the `px` width/height of the
```

Comment:
This is interesting, but ultimately more work and probably not needed. I think the string size matches better what we will need for designs.

Reply:
Yeah let's be more opinionated about this to start. If teams utilizing this component start to find that they can't use the component with some form of S,M,L, that's probably a good time to revisit

Reply:
Sounds good! I will remove the numeric option